api/db/activity.js

```diff
@@ -24,6 +24,17 @@ module.exports = () => ({
       return this.hasMany('apdActivitySchedule');
     },
 
+    static: {
+      withRelated: [
+        'approaches',
+        'goals',
+        'goals.objectives',
+        'expenses',
+        'expenses.entries',
+        'schedule'
+      ]
+    },
```

Comment:
If you put a `static` property on a model object, all of its properties will be attached to the model constructor.  They all get `modelName` for free (see below), but I figured this way we could easily add more stuff later if we needed/wanted.

Reply:
👌